packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -242,6 +242,7 @@ class Textarea extends React.Component {
     }
 
     additionalTextareaProps['aria-label'] = ariaLabelText;
+    additionalTextareaProps['aria-invalid'] = isInvalid;
```

Comment:
Can we add a Jest test case for the value of the 'aria-invalid' when `isInvalid` is true?